benchmarks/src/tpch.rs

```diff
@@ -439,7 +439,10 @@ fn col_to_scalar(column: &ArrayRef, row_index: usize) -> ScalarValue {
             ScalarValue::Decimal128(Some(array.value(row_index)), *p, *s)
         }
         DataType::Date32 => {
-            let array = column.as_any().downcast_ref::<Date32Array>().unwrap();
+            let array = match as_date32_array(column) {
+                Ok(array) => array,
+                Err(e) => panic!("{}", e),
+            };
```

Comment:
I think `unwrap` will also print the error message if called on an error:

https://doc.rust-lang.org/std/result/enum.Result.html#panics-1

> Panics if the value is an [Err](https://doc.rust-lang.org/std/result/enum.Result.html#variant.Err), with a panic message provided by the [Err](https://doc.rust-lang.org/std/result/enum.Result.html#variant.Err)’s value.


```suggestion
            let array = as_date32_array(column).unwrap();
```